cmd/frcli/utils.go

```diff
@@ -159,12 +159,23 @@ func extractPathArgs(ctx *cli.Context) (string, string, error) {
 	if faradayDir != faraday.FaradayDirBase ||
 		networkStr != faraday.DefaultNetwork {
 
-		tlsCertPath = filepath.Join(
-			faradayDir, networkStr, faraday.DefaultTLSCertFilename,
-		)
-		macPath = filepath.Join(
-			faradayDir, networkStr, faraday.DefaultMacaroonFilename,
-		)
+		// Only overwrite the tls cert path if the user has not
+		// explicitly defined it.
+		if !ctx.GlobalIsSet(tlsCertFlag.Name) {
```

Comment:
I guess the alternative way for detecting this is to compare the value against the default value (which I think is what we do in `lnd` and `litd`), which also works when using a config file. So if we ever add config file support to Faraday, this will likely break...

Reply:
ah - I thought config files are only for the long running daemons and not the CLIs right?

Reply:
but I can copy the pattern used in LNDs `extractPathArgs` if that will be better?

Reply:
Oh, you're absolutely right, I completely missed that part, didn't look at the file this change was made in... Okay, never mind then.